index.html

```diff
@@ -0,0 +1,67 @@
+<html lang="en">
```

Comment:
🔴 You also need to add a [doctype](https://developer.mozilla.org/en-US/docs/Glossary/Doctype) here:

```suggestion
<!DOCTYPE html>
<html lang="en">
```